network-api/networkapi/wagtailcustomization/static/wagtailadmin/css/custom-fix.css

```diff
@@ -17,49 +17,3 @@ body#wagtail
     height: unset;
   }
 }
-
-/*
-  To address bug from Wagtail v3
-  See context: https://github.com/mozilla/foundation.mozilla.org/issues/10194
-*/
-
-body#wagtail
-  main#main
-  form
-  ul
-  li
-  .field
-  .c-dropdown.t-inverted.c-dropdown--large
-  a {
-  background-color: var(--color-primary-darker);
-  color: white;
-}
-
-body#wagtail
-  main#main
-  form
-  ul
-  li
-  .field
-  .c-dropdown.t-inverted.c-dropdown--large
-  .c-dropdown__item
-  a {
-  background-color: transparent;
-}
-
-/*
```

Comment:
To test, 

- clone staging or prod db
- visit http://test.example.com:8000/cms/pages/47/edit/ on Firefox
- verify the bug is no longer there even after we remove the custom CSS here